client/src/components/onboarding/onboarding.scss

```diff
@@ -91,6 +91,13 @@ $padding: $spacer * 2;
   padding: 0 $padding;
 }
 
+.full-modal {
```

Comment:
what is this

Reply:
the extra div the eventhandler is listening to

Reply:
changed the classname